src/Intern_Budgethold.Infrastructure/DataAccess/AppDbContext.cs

```diff
@@ -0,0 +1,29 @@
+using Microsoft.EntityFrameworkCore;
+using Intern_Budgethold.Features.UserAuth;
+using Intern_Budgethold.Features.WalletManagement;
+
+namespace Intern_Budgethold.Infrastructure.DataAccess;
+
+public class AppDbContext : DbContext
+{
+  private readonly IEnumerable<IEntityTypeConfiguration> _configurations;
```

Comment:
Te TypeConfiguration można typować na konkretne encje, i z reguły jak już je mamy ztypowane to lepiej ręcznie je dodawać do OnModelCreating niż dynamicznie ładować